modules/nf-core/custom/getchromsizes/custom-getchromsizes.diff

```diff
@@ -0,0 +1,61 @@
+Changes in module 'nf-core/custom/getchromsizes'
+--- modules/nf-core/custom/getchromsizes/main.nf
++++ modules/nf-core/custom/getchromsizes/main.nf
+@@ -1,5 +1,5 @@
+ process CUSTOM_GETCHROMSIZES {
+-    tag "$fasta"
++    tag "$meta.id"
+     label 'process_single'
+ 
+     conda "bioconda::samtools=1.16.1"
+@@ -8,22 +8,29 @@
+         'biocontainers/samtools:1.16.1--h6899075_1' }"
+ 
+     input:
+-    tuple val(meta), path(fasta)
++    tuple   val(meta), path(fasta)
++    val     suffix
+ 
+     output:
+-    tuple val(meta), path ("*.sizes"), emit: sizes
+-    tuple val(meta), path ("*.fai")  , emit: fai
+-    tuple val(meta), path ("*.gzi")  , emit: gzi, optional: true
+-    path  "versions.yml"             , emit: versions
++    tuple val(meta), path ("*.${suffix}")   , emit: sizes
++    tuple val(meta), path ("*.fa")          , emit: fasta
++    tuple val(meta), path ("*.fai")         , emit: fai
++    tuple val(meta), path ("*.gzi")         , emit: gzi, optional: true
++    path  "versions.yml"                    , emit: versions
+ 
+     when:
+     task.ext.when == null || task.ext.when
+ 
+     script:
+-    def args = task.ext.args ?: ''
++    def args    = task.ext.args ?: ''
++    def prefix  = task.ext.prefix ?: "${meta.id}"
+     """
+-    samtools faidx $fasta
+-    cut -f 1,2 ${fasta}.fai > ${fasta}.sizes
++    samtools faidx $fasta -o ${prefix}.fa.fai
++    cut -f 1,2 ${prefix}.fa.fai > ${prefix}.${suffix}
++
++    if [[ "${fasta}" != "${prefix}-ref.fa" ]]; then
++        mv ${fasta} ${prefix}-ref.fa
```

Comment:
These are the output files I can see in my partial test
```
Oscheius_DF5033_1.fa.fai
Oscheius_DF5033_1.genome
Oscheius_DF5033_1-ref.fa
```

Just checking it's OK to have the Fasta file differ.

Reply:
We've not had an issue so far, I would prefer it to be the same. However, it wouldn't let me output the fasta without changing the name.

Reply:
See #133 for my solution